apps/watch/pages/fonts/fonts.css

```
@@ -0,0 +1,34 @@
+    url('/watch/assets/fonts/apercu_regular.svg') format('svg'),
+    url('/watch/assets/fonts/apercu_bold.otf');
+  font-weight: normal;
+  font-style: normal;
+  font-display: swap;
+}
+
+@font-face {
+  font-family: 'Montserrat';
+  src: url('./Montserrat-VariableFont_wght.ttf') format('woff2-variations');
+  src: url('./Montserrat-VariableFont_wght.ttf') format('woff2')
+    tech('variations');
```

Biome (1.9.4)

[error] Unexpected unknown function: tech

Use a known function instead.
See MDN web docs for more details.

(lint/correctness/noUnknownFunction)